tests/Fixer/ControlStructure/NoBreakCommentFixerTest.php

```diff
@@ -0,0 +1,504 @@
+            case 2:
+                bar();
+        }
+        break;
+    case 2:
+        switch ($bar) {
+            case 1:
+                bar();
+                // no break
+            case 2:
+                foo();
+        }
+}',
+                '<?php
+switch ($foo) {
+    case 1:
+        switch ($bar) {
+            case 1:
+                foo();
+            case 2:
+                bar();
+        }
+        break;
+    case 2:
+        switch ($bar) {
+            case 1:
+                bar();
+            case 2:
+                foo();
+        }
+}',
+            ),
+            array(
+                '<?php
+switch ($foo) {
+    case 1:
+        switch ($bar):
+            case 1:
+                foo();
+                // no break
+            case 2:
+                bar();
+        endswitch;
+        break;
+    case 2:
+        switch ($bar):
+            case 1:
+                bar();
+                // no break
+            case 2:
+                foo();
+        endswitch;
+}',
+                '<?php
+switch ($foo) {
+    case 1:
+        switch ($bar):
+            case 1:
+                foo();
+            case 2:
+                bar();
+        endswitch;
+        break;
+    case 2:
+        switch ($bar):
+            case 1:
+                bar();
+            case 2:
+                foo();
+        endswitch;
+}',
+            ),
+            array(
+                '<?php
+switch ($foo) {
+    case 1:
+        foo();
+        continue;
+    case 2:
+        bar();
+        continue;
+    default:
+        baz();
+}',
+                '<?php
+switch ($foo) {
+    case 1:
+        foo();
+        // no break
+        continue;
+    case 2:
+        bar();
+        // no break
+        continue;
+    default:
+        baz();
+}',
+            ),
+            array(
+                '<?php
+switch ($foo) {
+    case 1:
+        return foo();
+    case 2:
+        return bar();
+    default:
+        return baz();
+}',
+                '<?php
+switch ($foo) {
+    case 1:
+        // no break
+        return foo();
+    case 2:
+        // no break
+        return bar();
+    default:
+        return baz();
+}',
+            ),
+            array(
+                '<?php
+switch ($foo) {
+    case 1:
+        foo();
+        break;
+    case 2:
+        bar();
+        break;
+    default:
+        baz();
+}',
+                '<?php
+switch ($foo) {
+    case 1:
+        foo();
+        // no break
+        break;
+    case 2:
+        bar();
+        // no break
+        break;
+    default:
+        baz();
+}',
+            ),
+            array(
+                '<?php
+switch ($foo) {
+    case 1:
+    case 2:
+        bar();
+        break;
+    default:
+        baz();
+}',
+                '<?php
+switch ($foo) {
+    case 1:
+        // no break
+    case 2:
+        bar();
+        // no break
+        break;
+    default:
+        baz();
+}',
+            ),
```

Comment:
please add a test with at least 3 `switch` constructions all with missing comments and make all of those on one line